.github/workflows/agw.branch_push.yml

```diff
@@ -12,7 +13,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Install tox
-        run: pip install tox
+        run: pip install tox==3.21.4
```

Comment:
Maybe instead of defining the version everywhere, we can define it in one place?

Reply:
Same for all workflows changed in this PR

Reply:
I found a cool solution, using a custom GH action, which allows you storing variables in a single file. You need to source the file in every job and every workflow separately, but at least it's single source of truth.

Reply:
Nice. 
I will use it :)